src/components/views/right_panel/RoomHeaderButtons.tsx

```diff
@@ -64,14 +64,14 @@ const UnreadIndicator: React.FC<IUnreadIndicatorProps> = ({ color }) => {
 
     const classes = classNames({
         mx_Indicator: true,
-        mx_RoomHeader_button_unreadIndicator: true,
+        mx_LegacyRoomHeader_button_unreadIndicator: true,
```

Comment:
This implies this should be LegacyRoomHeaderButtons

Reply:
Sounds good, will update the files name. You're right.